### DISCUSSION TITLE
How can multiple IDs of one application be managed in WinGet?

### DISCUSSION THREAD

**Discussion by rwiesbach** [microsoft/winget-cli#3630] [Q&A] [answered]
Yesterday I noticed that RealVNC Viewer has ID RealVNC.VNCViewer on the repo, but the installed version (`winget list`) shows a GUID as identifier (installed without winget). The result is that `winget list --upgrade-available` does not show the update even though it is on the repo (loacl 7.5.0, repo 7.5.1)

What can be done about this?

Thanks

**Comment by Trenly** [comment, marked as answer]
We would have to add the previous version (7.5.0) to the winget-pkgs repo, or would need to look at the metadata of both to see if the manifest for 7.5.1 can be updated to provide a better match

**Comment by rwiesbach** [comment]
So I need to report this to -pkgs not to -cli?

**Reply by denelon** [reply]
Yes, you could ask for the earlier version of the package to be added to the winget-pkgs repository.
